chia/cmds/plotnft_funcs.py

```diff
@@ -270,106 +274,133 @@ async def submit_tx_with_confirmation(
         print(f"Error performing operation on Plot NFT -f {fingerprint} wallet id: {wallet_id}: {e}")
 
 
+async def wallet_id_lookup_and_check(wallet_client: WalletRpcClient, wallet_id: Optional[int]) -> int:
+    selected_wallet_id: int
+    try:
+        pool_wallets = await wallet_client.get_wallets(wallet_type=WalletType.POOLING_WALLET)
+    except ValueError as e:
```

Comment:
What are you expecting to raise a `ValueError`?

https://github.com/Chia-Network/chia-blockchain/blob/dd028574bb8cbf7526194f3a0be5ee83926d13c9/chia/rpc/rpc_client.py#L109-L117

`"success": false` will trigger `ResponseFailureError`.  I would expect most other errors from that fetch code would be networking or json specific.

Reply:
Hmm, I guess we (I?) made `ResponseFailureError` inherit from `ValueError` for backwards compatibility...  so yeah, you'll catch that.  But if that's the intent then maybe we should use that name `:]`

Reply:
I removed this try handler - I took a look and this call really won't fail absent network errors so I dont't think this needs a specific handler. In the strange case where it might raise ResponseFailureError, that doesn't need any special handling here